extensions/renderer/xwalk_module_system.cc

```diff
@@ -357,6 +357,7 @@ void XWalkModuleSystem::DeleteExtensionModules() {
 void XWalkModuleSystem::LoadExtensionForTrampoline(
     v8::Isolate* isolate,
     v8::Local<v8::Value> data) {
+  v8::HandleScope handle_scope(isolate);
```

Comment:
It create a sub HandleScope here instead of using the default scope, please refer the comments , https://code.google.com/p/chromium/codesearch#chromium/src/v8/src/handles.h&q=HandleScope&sq=package:chromium&type=cs&l=170